exercises/concept/recycling-robot/.meta/exemplar.js

```diff
@@ -0,0 +1,130 @@
+// @ts-check
+//
+// The line above enables type checking for this file. Various IDEs interpret
+// the @ts-check directive. It will give you helpful autocompletion when
+// implementing this exercise.
+
+export class ElectronicDevice {
+  // This class will be used in the exercise.
+}
+/**
+ * Checks if input is a boolean.
+ *
+ * @param {any} value
+ * @returns {boolean} whether the input is a boolean
+ */
+export function isBoolean(value) {
+  return typeof value === 'boolean';
+}
+
+/**
+ * Checks if input is a finite number or bigint.
+ *
+ * @param {any} value
+ * @returns {boolean} whether the input is a finite number or bigint
```

Comment:
```suggestion
 * @returns {value is number | bigint} whether the input is a finite number or bigint
```

Dunno if this works; can you see if this gives any type hints in vscode?